### ISSUE TITLE
"Script path not specified." with spoon-runner-1.3.0

### ISSUE THREAD

**Issue by gastoncesarf** [square/spoon#309]
When i am trying to execute this:
```
java -jar spoon-runner-1.3.0-jar-with-dependencies.jar \
    --apk myApp.apk \
    --test-apk myAppdebug-androidTest-unaligned.apk
```
shows "Script path not specified". That did not happen with version 1.2.1.

**Comment by jraska** [comment]
+1

**Comment by edenman** [comment]
@gastoncesarf @jraska thanks for the report.  Fixed in 1.3.1 which I just released, should be live in an hour or two.
